enterprise/wsproxy/wsproxy.go

```diff
@@ -235,8 +235,8 @@ func New(ctx context.Context, opts *Options) (*Server, error) {
 	r.Get("/healthz-report", s.healthReport)
 	r.NotFound(func(rw http.ResponseWriter, r *http.Request) {
 		site.RenderStaticErrorPage(rw, r, site.ErrorPageData{
-			Status:       404,
-			Title:        "Route Not Found",
+			Status:       http.StatusBadRequest,
```

Comment:
```suggestion
			Status:       http.StatusBadRequest,
```
We shouldn't show this error code in the page, it's not helpful to the user.

Reply:
I'll mess with the template.